server/users/users.go

```diff
@@ -0,0 +1,52 @@
+package users
+
+import (
+	"github.com/argoproj/argo-cd/util/password"
+	"github.com/argoproj/argo-cd/util/session"
+	"github.com/argoproj/argo-cd/util/settings"
+	"golang.org/x/net/context"
+)
+
+// Server provides a Session service
+type Server struct {
+	sessionMgr  *session.SessionManager
+	settingsMgr *settings.SettingsManager
+}
+
+// NewServer returns a new instance of the Session service
+func NewServer(sessionMgr *session.SessionManager, settingsMgr *settings.SettingsManager) *Server {
+	return &Server{
+		sessionMgr:  sessionMgr,
+		settingsMgr: settingsMgr,
+	}
+
+}
+
+//This Function is used to Update User Passwords \
```

Comment:
Go doc convention is to repeat the name of the function in the comment. e.g.:

// UpdatePassword updates a user's password